### PR TITLE
HoC 2023 - Create hourofcode.com/ai page

### PR DESCRIPTION
**Pull request by kelbyhawn**
Create https://hourofcode.com/ai page
- Page is fully responsive

**Jira ticket:** [ACQ-1049](https://codedotorg.atlassian.net/browse/ACQ-1049)

----

Artboard

### CODE REVIEW

File: pegasus/sites.v3/hourofcode.com/views/carousel_pagination.haml
Comment (kelbyhawn) [inline pegasus/sites.v3/hourofcode.com/views/carousel_pagination.haml:1]: Created reusable pagination view for the carousels.

File: pegasus/sites.v3/hourofcode.com/public/css/page/how-to.scss
Comment (kelbyhawn) [inline pegasus/sites.v3/hourofcode.com/public/css/page/how-to.scss:1]: I refactored these css changes to be more reusable.

File: pegasus/sites.v3/hourofcode.com/public/css/action-blocks.scss
Comment (kelbyhawn) [inline pegasus/sites.v3/hourofcode.com/public/css/action-blocks.scss:1]: Copied over from the [design-system-pegasus.scss stylesheet](https://github.com/code-dot-org/code-dot-org/blob/9410b16b7fcdba6d1f9a7301bf370141659df30a/pegasus/sites.v3/code.org/public/css/design-system-pegasus.scss#L630-L764) — will consider refactoring all of this to be shareable across code.org and hourofcode.com sites when we aren't in a time crunch.

**Review by TurnerRiley — APPROVED** [review]
This looks great! Thanks for marking spots for future refactoring when there's less of a time crunch

**Pull request merged**